conf/output/test-output-0.json

```diff
@@ -3,7 +3,6 @@
   "type" : [ "BibliographicResource", "Article" ],
   "rpbId" : "929t112200",
   "id" : "https://lobid.org/resources/929t112200",
-  "hbzId" : "CT003010348",
```

Comment:
This seems to be the real hbzId of the resource not the one of the parent publication.